app/src/main/java/com/example/codingquiz/ui/screen/StatsScreen.kt

```diff
@@ -0,0 +1,194 @@
+package com.example.codingquiz.ui.screen
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.codingquiz.R
+import com.example.codingquiz.data.domain.AnsweredQuestionsCountStats
+import com.example.codingquiz.data.domain.CategoryStats
+import com.example.codingquiz.data.domain.CorrectAnswersStats
+import com.example.codingquiz.ui.common.HeaderTextLarge
+import com.example.codingquiz.ui.common.HeaderTextMedium
+import com.example.codingquiz.ui.common.TwoTextsRow
+import com.example.codingquiz.ui.theme.CodingQuizTheme
+import com.example.codingquiz.viewmodel.StatsViewModel
+import org.koin.androidx.compose.koinViewModel
+
+@Composable
+fun StatsScreen(
+    statsViewModel: StatsViewModel = koinViewModel()
+) {
+    Column {
+        LaunchedEffect(Unit) {
+            statsViewModel.getMostAnsweredCategories()
+            statsViewModel.getAnsweredQuestionsCount()
+            statsViewModel.getCorrectAnswersCount()
+        }
+
+        Column(
+            modifier = Modifier.padding(12.dp),
+            verticalArrangement = Arrangement.spacedBy(20.dp),
+        ) {
+            val categoryStats by statsViewModel.categoryStats.collectAsState()
```

Comment:
Shouldn't it be higher that the main Column (outside, top-level inside this function)?